deptrack-devops-recipes/src/Devops/Bootstrap/Parasite.hs

```diff
@@ -184,3 +184,34 @@ remotedWith mkAction usr clo host =
                                 , "-l", unpack (userName u), unpack ip
                                 , remoteExecution rpath b64 ]
         remoteExecution rpath b64 = unwords [ "chmod", "+x", rpath, ";", rpath, magicRemoteArgv, unpack b64 , ";"]
+
+
+-- | Remotely callback some `Continued a` on some parasited host.
+--
+remoteContinued :: (Typeable a)
+               => DevOp User -> Continued a -> DevOp ParasitedHost -> DevOp (Remoted a)
+remoteContinued usr cont host =
+  devop fst mkOp (do
+              u <- usr
+              c <- ssh
+              let obj = eval cont
+                  (BinaryCall _ fArgs) = callback cont
+                  args = fArgs SequentialTurnUp
+              (ParasitedHost rpath _ r) <- host
+              return (Remoted r obj,(rpath, c, u,args,  r)))
+
+  where ssh = binary :: DevOp (Binary "ssh")
```

Comment:
break the line after `where` and 4-chars indent the code. one goal is that the `where` stands out and the where-defined symbols align with the code above where it's used